lab_1/main.py

```diff
@@ -4,14 +4,23 @@
 """
 
 
+import re
+
+
 def tokenize(text: str) -> list or None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    else:
```

Comment:
do you need this `else`?

Reply:
No, this else is not needed here. I will delete it.